### DISCUSSION TITLE
Your Azure Static Web App is live and waiting for your content error

### DISCUSSION THREAD

**Discussion by DubeyTarun15** [microsoft/SATechnicalOnboarding#61] [Questions]
i tried more than 10 times , did every possible thing but the error is still there like i followed 2 different tutorial also for this , every time same error , i have proper credits also . what should i do  ???
![image_2023-10-02_133612522](https://github.com/microsoft/SATechnicalOnboarding/assets/121306930/168de954-d18e-4494-a013-2ba3aa9a1a81)
![image_2023-10-02_133644371](https://github.com/microsoft/SATechnicalOnboarding/assets/121306930/d18da676-ddae-464f-8602-695b9ec2ee55)

**Comment by gabrielpiauilino-mlsa** [comment]
Hello @DubeyTarun15, how are you?
Please, which repository, for example, are you using?

**Reply by Habramh5** [reply]
I'm facing the same problem, and I'm really stuck. Help please!

**Comment by Habramh5** [comment]
I'm facing the same problem, and I'm really stuck. Help please!

**Reply by AnthonyByansi** [reply]
Hi @Habramh5 
To ensure a seamless build process, it's important to verify that the directory path for your application setup is correctly configured.

Please refer to the attached screenshot for step-by-step guidance on setting up the appropriate directory path for your application build. Should you encounter any challenges or require further assistance, feel free to reach out for immediate support. and here is a full [Azure static web app Guide](https://learn.microsoft.com/azure/static-web-apps/getting-started?tabs=vanilla-javascript&WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352)
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/fe053d11-fd07-4e00-871b-66c0b8a7165c)
